quantum/rgblight.h

```diff
@@ -139,6 +139,27 @@ enum RGBLIGHT_EFFECT_MODE {
 #        include <avr/pgmspace.h>
 #    endif
 
+#    ifdef RGBLIGHT_LAYERS
+typedef struct {
+    uint8_t index;      // The first LED to light
+    uint8_t count;      // The number of LEDs to light
+    uint8_t h;
+    uint8_t s;
+    uint8_t v;
```

Comment:
// In every other struct hue, sat, val are used to name h, s and v
    uint8_t hue;
    uint8_t sat;
    uint8_t val;

Reply:
Good catch. I meant to do that earlier but it slipped my mind.